src/frontend/components/pages/shoppingList/shopping-list.service.ts

```diff
@@ -0,0 +1,74 @@
+import {Injectable} from '@angular/core';
+import {Observable} from 'rxjs/Rx';
+import {Http} from "@angular/http";
+import {AppService} from "../../app.service";
+import {Article} from "../../../models/Article";
+
+@Injectable()
+export class ShoppingListService extends AppService {
```

Comment:
Sehr viele `any`-Types in dieser Klasse... Könnte man hier vlt. mehr typisieren?
`private groupObj: any = {groups: []};`
`public articles$: Observable<any>`
etc.